2021-04-10 10:12:10,213 main.py:30  INFO Creating a new UserCollection
2021-04-10 10:12:10,215 main.py:39  INFO Createing a new UserStatusCollection
2021-04-10 10:12:10,223 main.py:211 INFO Adding a new User
2021-04-10 10:12:10,224 main.py:223 INFO Updating an User
2021-04-10 10:12:10,225 main.py:236 INFO Deleting an User
2021-04-10 10:12:10,227 main.py:250 INFO Searching for an User
2021-04-10 10:12:10,228 main.py:250 INFO Searching for an User
2021-04-10 10:12:10,229 main.py:268 INFO Adding an UserStatus
2021-04-10 10:12:10,230 main.py:280 INFO Updating an UserStatus
2021-04-10 10:12:10,232 main.py:292 INFO Deleting an UserStatus
2021-04-10 10:12:10,233 main.py:305 INFO Searching for an UserStatus
2021-04-10 10:12:10,235 main.py:305 INFO Searching for an UserStatus
2021-04-10 10:12:10,236 main.py:316 INFO Searching for matching UserStatuses
2021-04-10 10:12:10,237 main.py:325 INFO Searching for UserStatuses matching string
2021-04-10 10:12:10,899 main.py:30  INFO Creating a new UserCollection
2021-04-10 10:12:10,981 users.py:27  INFO New UserCollection instance created
2021-04-10 10:12:10,981 main.py:39  INFO Createing a new UserStatusCollection
2021-04-10 10:12:11,113 user_status.py:23  INFO New UserStatusCollection instance created
2021-04-10 10:12:11,131 main.py:66  INFO Loading users from accounts.csv
2021-04-10 10:12:11,575 main.py:108 INFO Saving collection in database
2021-04-10 10:12:11,579 main.py:108 INFO Saving collection in database
2021-04-10 10:12:12,139 main.py:211 INFO Adding a new User
2021-04-10 10:12:12,158 users.py:45  INFO User dave03 sucessfully added
2021-04-10 10:12:12,159 main.py:211 INFO Adding a new User
2021-04-10 10:12:12,164 users.py:43  ERROR User dave03 breaks database rule: E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }, full error: {'index': 0, 'code': 11000, 'keyPattern': {'user_id': 1}, 'keyValue': {'user_id': 'dave03'}, 'errmsg': 'E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }'}
2021-04-10 10:12:12,164 main.py:223 INFO Updating an User
2021-04-10 10:12:12,182 users.py:65  INFO User dave03 sucessfully modified
2021-04-10 10:12:12,182 main.py:223 INFO Updating an User
2021-04-10 10:12:12,184 users.py:61  ERROR User dave02 not found in database
2021-04-10 10:12:12,184 main.py:250 INFO Searching for an User
2021-04-10 10:12:12,186 users.py:96  INFO User dave03 sucessfully found
2021-04-10 10:12:12,186 main.py:250 INFO Searching for an User
2021-04-10 10:12:12,188 users.py:94  ERROR User dave02 not found in database
2021-04-10 10:12:12,188 main.py:236 INFO Deleting an User
2021-04-10 10:12:12,197 users.py:83  INFO User dave03 sucessfully deleted
2021-04-10 10:12:12,197 main.py:236 INFO Deleting an User
2021-04-10 10:12:12,199 users.py:78  ERROR User dave03 not found in database
2021-04-10 10:12:12,199 main.py:211 INFO Adding a new User
2021-04-10 10:12:12,201 users.py:45  INFO User dave03 sucessfully added
2021-04-10 10:12:12,216 main.py:145 INFO Loading users from status_updates.csv
2021-04-10 10:12:14,423 main.py:187 INFO Saving collection in database
2021-04-10 10:12:14,483 main.py:187 INFO Saving collection in database
2021-04-10 10:12:14,545 main.py:187 INFO Saving collection in database
2021-04-10 10:12:14,600 main.py:187 INFO Saving collection in database
2021-04-10 10:12:14,626 main.py:187 INFO Saving collection in database
2021-04-10 10:12:14,636 main.py:187 INFO Saving collection in database
2021-04-10 10:12:14,667 main.py:187 INFO Saving collection in database
2021-04-10 10:12:14,720 main.py:187 INFO Saving collection in database
2021-04-10 10:12:14,739 main.py:187 INFO Saving collection in database
2021-04-10 10:12:14,747 main.py:187 INFO Saving collection in database
2021-04-10 10:12:14,803 main.py:187 INFO Saving collection in database
2021-04-10 10:12:14,868 main.py:187 INFO Saving collection in database
2021-04-10 10:12:16,390 main.py:268 INFO Adding an UserStatus
2021-04-10 10:12:16,397 user_status.py:45  INFO Status dave03_01 added to database
2021-04-10 10:12:16,397 main.py:268 INFO Adding an UserStatus
2021-04-10 10:12:16,400 user_status.py:43  ERROR Status dave03_01 already exists in database
2021-04-10 10:12:16,401 main.py:280 INFO Updating an UserStatus
2021-04-10 10:12:16,403 user_status.py:62  INFO Status dave03_01 sucessfully modified
2021-04-10 10:12:16,403 main.py:280 INFO Updating an UserStatus
2021-04-10 10:12:16,405 user_status.py:57  ERROR Status dave03_02 not found database
2021-04-10 10:12:16,405 main.py:305 INFO Searching for an UserStatus
2021-04-10 10:12:16,408 user_status.py:90  INFO Status dave03_01 sucessfully found
2021-04-10 10:12:16,408 main.py:305 INFO Searching for an UserStatus
2021-04-10 10:12:16,410 user_status.py:88  ERROR Status dave03 not found database
2021-04-10 10:12:16,410 main.py:292 INFO Deleting an UserStatus
2021-04-10 10:12:16,413 user_status.py:76  INFO Status dave03_01 sucessfully deleted
2021-04-10 10:12:16,413 main.py:292 INFO Deleting an UserStatus
2021-04-10 10:12:16,415 user_status.py:74  ERROR Status dave03_01 not found database
2021-04-10 10:12:16,415 main.py:268 INFO Adding an UserStatus
2021-04-10 10:12:16,418 user_status.py:45  INFO Status dave03_01 added to database
2021-04-10 10:12:16,419 main.py:236 INFO Deleting an User
2021-04-10 10:12:16,516 users.py:83  INFO User dave03 sucessfully deleted
2021-04-10 10:12:16,516 main.py:305 INFO Searching for an UserStatus
2021-04-10 10:12:16,518 user_status.py:88  ERROR Status dave03_01 not found database
2021-04-10 10:12:16,519 main.py:211 INFO Adding a new User
2021-04-10 10:12:16,519 users.py:105 ERROR User id ############################## is longer than 30
2021-04-10 10:12:16,519 main.py:211 INFO Adding a new User
2021-04-10 10:12:16,519 users.py:109 ERROR User name ############################## is longer than 30
2021-04-10 10:12:16,519 main.py:211 INFO Adding a new User
2021-04-10 10:12:16,519 users.py:113 ERROR User last name #################################################################################################### is longer than 100
2021-04-10 10:12:16,522 user_status.py:23  INFO New UserStatusCollection instance created
2021-04-10 10:12:16,524 user_status.py:23  INFO New UserStatusCollection instance created
2021-04-10 10:12:16,525 user_status.py:45  INFO Status rando_001 added to database
2021-04-10 10:12:16,542 user_status.py:23  INFO New UserStatusCollection instance created
2021-04-10 10:12:16,543 user_status.py:43  ERROR Status rando_001 already exists in database
2021-04-10 10:12:16,545 user_status.py:23  INFO New UserStatusCollection instance created
2021-04-10 10:12:16,546 user_status.py:39  ERROR User rando does not exist
2021-04-10 10:12:16,547 user_status.py:23  INFO New UserStatusCollection instance created
2021-04-10 10:12:16,548 user_status.py:62  INFO Status rando_001 sucessfully modified
2021-04-10 10:12:16,550 user_status.py:23  INFO New UserStatusCollection instance created
2021-04-10 10:12:16,551 user_status.py:57  ERROR Status rando_001 not found database
2021-04-10 10:12:16,552 user_status.py:23  INFO New UserStatusCollection instance created
2021-04-10 10:12:16,553 user_status.py:76  INFO Status rando_001 sucessfully deleted
2021-04-10 10:12:16,555 user_status.py:23  INFO New UserStatusCollection instance created
2021-04-10 10:12:16,555 user_status.py:74  ERROR Status rando_001 not found database
2021-04-10 10:12:16,557 user_status.py:23  INFO New UserStatusCollection instance created
2021-04-10 10:12:16,557 user_status.py:90  INFO Status rando_001 sucessfully found
2021-04-10 10:12:16,559 user_status.py:23  INFO New UserStatusCollection instance created
2021-04-10 10:12:16,560 user_status.py:88  ERROR Status rando_001 not found database
2021-04-10 10:12:16,561 user_status.py:23  INFO New UserStatusCollection instance created
2021-04-10 10:12:16,562 user_status.py:99  INFO Found statuses posted by rando
2021-04-10 10:12:16,564 user_status.py:23  INFO New UserStatusCollection instance created
2021-04-10 10:12:16,564 user_status.py:107 INFO Found statuses matching best
2021-04-10 10:12:16,567 users.py:27  INFO New UserCollection instance created
2021-04-10 10:12:16,569 users.py:27  INFO New UserCollection instance created
2021-04-10 10:12:16,569 users.py:45  INFO User rando sucessfully added
2021-04-10 10:12:16,571 users.py:27  INFO New UserCollection instance created
2021-04-10 10:12:16,571 users.py:43  ERROR User rando breaks database rule: ERR_MSG
2021-04-10 10:12:16,573 users.py:27  INFO New UserCollection instance created
2021-04-10 10:12:16,574 users.py:105 ERROR User id ############################## is longer than 30
2021-04-10 10:12:16,575 users.py:27  INFO New UserCollection instance created
2021-04-10 10:12:16,576 users.py:65  INFO User rando sucessfully modified
2021-04-10 10:12:16,577 users.py:27  INFO New UserCollection instance created
2021-04-10 10:12:16,578 users.py:61  ERROR User rando not found in database
2021-04-10 10:12:16,579 users.py:27  INFO New UserCollection instance created
2021-04-10 10:12:16,580 users.py:105 ERROR User id ############################## is longer than 30
2021-04-10 10:12:16,582 users.py:27  INFO New UserCollection instance created
2021-04-10 10:12:16,583 users.py:83  INFO User rando sucessfully deleted
2021-04-10 10:12:16,584 users.py:27  INFO New UserCollection instance created
2021-04-10 10:12:16,585 users.py:78  ERROR User rando not found in database
2021-04-10 10:12:16,586 users.py:27  INFO New UserCollection instance created
2021-04-10 10:12:16,587 users.py:96  INFO User rando sucessfully found
2021-04-10 10:12:16,589 users.py:27  INFO New UserCollection instance created
2021-04-10 10:12:16,589 users.py:94  ERROR User rando not found in database
2021-04-10 10:12:16,591 users.py:27  INFO New UserCollection instance created
2021-04-10 10:12:16,591 users.py:105 ERROR User id ############################## is longer than 30
2021-04-10 10:12:16,591 users.py:109 ERROR User name ############################## is longer than 30
2021-04-10 10:12:16,591 users.py:113 ERROR User last name #################################################################################################### is longer than 100
